cmd/main.go

```diff
@@ -25,6 +24,10 @@ const (
 )
 
 func main() {
+	_, err := maxprocs.Set()
```

Comment:
I think the goal is to have 2 different behavior. 1 for the CLI (no logging) and 1 for the "services" (info logs).